Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -631,13 +634,9 @@ - (void)writeObjectToDefaults:(NSString *)key value:(NSObject *)value {
 
 - (void)persistPrefsToDisk {
     @synchronized (self) {
-        NSDictionary *persistenceDict = [self.persistenceDict copy];
+        if (!self.persistenceDict) return;
```

Comment:
Can you at least wrap this in a try/catch just in case? I looked through past commits and can't remember when we added it but I'm having terrible flashbacks to the late night crash reports coming in ~ a year ago for concurrent modification errors.